package.json

```
@@ -116,7 +118,10 @@
     "@folio/stripes-final-form": "^6.1.0",
     "@folio/stripes-form": "^7.1.0",
     "@folio/stripes-smart-components": "^7.2.0",
+    "@folio/stripes-testing": "^4.4.1000005196",
```

```suggestion
    "@folio/stripes-testing": "^4.4.1",
```
The CI will pull in snapshot automatically; we should use the release versions here